clients/fides-js/src/lib/tcf/vendors.ts

```diff
@@ -17,6 +22,20 @@ export const vendorIsGvl = (
   return gvl.vendors[vendor.id];
 };
 
+/**
+ * Given a vendor id such as `gvl.2`, return {source: "gvl", id: "2"};
+ */
+export const decodeVendorId = (vendorId: TCFVendorRelationships["id"]) => {
```

Comment:
can we rename this to `createSourceToVendorIdMap` or `splitSourceAndVendorId` or something?